tools/install-stack.sh

```diff
@@ -23,13 +23,15 @@ fetch_stack_windows() {
 
 # We need stack to generate cabal files with precise bounds, even for cabal
 # builds.
+pwd
 mkdir -p ~/.local/bin
 if [ "$(uname)" = "Darwin" ]; then
   retry fetch_stack_osx
 elif [ "$(uname)" = "Linux" ]; then
   retry fetch_stack_linux
 else
   retry fetch_stack_windows
+  choco install haskell-dev
```

Comment:
`choco` is not available on all Windows systems.  The script should only run this command if `$TRAVIS` env variable is defined.